pkg/reconciler/openshift/tektonconfig/extension.go

```diff
@@ -67,10 +68,37 @@ func (oe openshiftExtension) PreReconcile(ctx context.Context, tc v1alpha1.Tekto
 	r := rbac{
 		kubeClientSet:     oe.kubeClientSet,
 		operatorClientSet: oe.operatorClientSet,
-		ownerRef:          configOwnerRef(tc),
 		version:           os.Getenv(versionKey),
+		tektonConfig:      config,
 	}
-	return r.createResources(ctx)
+
+	var createResourceError error
```

Comment:
could we move this to the top or to the rbac package.
let us make this more descriptive/meaningful as well.